_includes/blog_header.html

```diff
@@ -0,0 +1,79 @@
+<div class="search-tool"
+      style="position: fixed; top: 0px ; bottom: 0px; left: 0px; right:  0px; opacity: 0.95; background-color: #111111; z-index: 9999; display: none;">
```

Comment:
This is **the** search tool provided by the template. We may upgrade it in further iterations.